gpu/src/wgpu_impl/shaders/alpha_triangles.wgsl

```diff
@@ -0,0 +1,25 @@
+struct VertexInput {
+  @location(0) pos: vec2<f32>,
+  @location(1) value: f32,
+}
+
+
+struct VertexOutput {
+  @builtin(position) pos: vec4<f32>,
+  @location(0) value: f32,
+}
+
+
+@vertex
+fn vs_main(input: VertexInput) -> VertexOutput {
+    let pos = input.pos * vec2(2., -2.) + vec2(-1., 1.);
```

Comment:
What meaning of these numbers? 😂

Reply:
// convert from gpu-backend coords(0..1) to wgpu corrds(-1..1)